api/environments/local

```diff
@@ -16,7 +16,7 @@ NOMAD_HOST=nomad
 LOCAL_ROOT_DIR=/home/user/data_store
 USE_S3=False
 S3_BUCKET_NAME=data-refinery
-S3_QN_TARGET_BUCKET_NAME=
+S3_QN_TARGET_BUCKET_NAME=data-refinery
```

Comment:
Why are you setting this? Local should not upload files to S3.

Reply:
I just needed it to have some value, without it, some tests were failing because they couldn't find this variable. I'm gonna try to find another way to resolve it.

Reply:
Ah ok. Maybe just an issue with the way the env var was being retrieved then.